src/app/views/datasets-new/create/CreateVersionController.jsx

```diff
@@ -167,7 +167,7 @@ export class CreateVersionController extends Component {
             .then(instances => {
                 this.setState({
                     isFetchingInstances: false,
-                    instances: this.mapInstancesToState(instances.items)
+                    instances: this.mapInstancesToState(instances.items.reverse())
```

Comment:
Looks like it will work so I will approve, but I just had a thought.  It might be more resilient to add a sort to this so that it will always sort based on the date.  That way Florence is not dependent on any changes to the dataset API.